source/includes/_endpoint-tabbook.md

```diff
@@ -1,24 +1,46 @@
 ## Tab books
 
-The `tabbook` view of a multitable will generate an excel (.xlsx) workbook containing each variable in the dataset crosstabbed with a given multitable. 
+The `tabbook` view of a multitable will generate an excel (.xlsx) workbook 
+containing each variable in the dataset crosstabbed with a given multitable. 
 
-A request to `datasets/{id}/multitables/{id}/tabbook/' will generate a download location to which the exporter will write this file when it is done computing (it may take some time for large datasets). Clients should note the download url, monitor progress, and when complete, GET the download location. 
+A request to `datasets/{id}/multitables/{id}/tabbook/' will generate a download 
+location to which the exporter will write this file when it is done computing 
+(it may take some time for large datasets). Clients should note the download url, 
+monitor progress, and when complete, POST the download location.
+ 
+The server will return a 202 response indicating that the export job started with
+a Location header indicating where the final exported file will be available.
+
+The response's body will contain the URL for the progress url where to query
+the state of the export job.
+
+Requesting the same job, if still in progress will return the same response 
+indicating the original progress to check. If the export is finished, the server
+will 302 redirect to the destination for download.
+
+If there have been changes on the dataset attributes, a new tabbook will be
+generated regardless of the status of any other pending exports.
+ 
+
+#### POST
+
+```http
+POST /api/datasets/{id}/multitables/{id}/tabbook/ HTTP/1.1
+```
+
+----
 
-#### GET
 
 ```http
-GET /api/datasets/{id}/multitables/{id}/tabbook/ HTTP/1.1
+HTTP/1.1 202 Accepted
+Location: https://s3-url/filename.xlsx
 ```
 
 ```json
 {
     "element": "shoji:view",
     "self": "/api/datasets/{id}/multitables/{id}/tabbook/",
-    "value": {
-        "status": "started",
-        "url": "https://s3-url/filename.xlsx",
-        "progress": "/api/progress/:{id}/"
-    }
+    "value": "/api/progress/{id}/"
```

Comment:
Disambiguate the `{id}`s please. Surely all three instances don't refer to the same `id`.